free-courses-en.md

```diff
@@ -180,8 +186,14 @@
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046)
 
 
+### JQuery
```

Comment:
I think there's some benefit to mimicking the subject index structure found in free-programming-books.md 

There, the jQuery resources list got so long that they were split into a separate file, so maybe this heading and the ###JS should be structured like

* [JavaScript](#javascript)
  * [Javascript frameworks](#javascript-frameworks)
  * [Node.js](#nodejs)